src/fides/api/service/privacy_request/request_runner_service.py

```diff
@@ -450,6 +452,11 @@ def run_privacy_request(
                     for dataset_config in datasets
                     if not dataset_config.connection_config.disabled
                 ]
+
+                # Add manual task artificial graphs to dataset graphs
+                manual_task_graphs = create_manual_task_artificial_graphs(session)
+                dataset_graphs.extend(manual_task_graphs)
```

Comment:
Creates graphs on the fly so the manual tasks can be represented as collections, and the manual task fields as dataset fields. This lets us treat the manual inputs as collection data.